packages/web/src/components/edit-folder-modal/EditFolderModal.tsx

```diff
@@ -60,30 +74,57 @@ const EditFolderModal = () => {
     [dispatch, folder, folderId, handleClose, playlistLibrary]
   )
 
+  const handleClickDelete = useCallback(() => {
+    setShowDeleteConfirmation(true)
+  }, [])
+
+  const handleConfirmDelete = useCallback(() => {
+    if (!(playlistLibrary == null || folderId == null || folder == null)) {
+      const newLibrary = removePlaylistFolderInLibrary(
+        playlistLibrary,
+        folderId
+      )
+      setShowDeleteConfirmation(false)
+      dispatch(updatePlaylistLibrary({ playlistLibrary: newLibrary }))
+    }
+    handleClose()
+  }, [dispatch, folder, folderId, handleClose, playlistLibrary])
+
   return (
-    <Modal
-      modalKey='editfolder'
-      isOpen={isOpen}
-      onClose={handleClose}
-      zIndex={zIndex.CREATE_PLAYLIST_MODAL}
-      bodyClassName={styles.modalBody}
-    >
-      <ModalHeader onClose={handleClose}>
-        <ModalTitle
-          icon={<IconFolder />}
-          title={messages.editPlaylistFolderModalTitle}
-        />
-      </ModalHeader>
-      <ModalContent>
-        <FolderForm
-          isEditMode
-          onSubmit={handleSubmit}
-          onCancel={handleClose}
-          onDelete={() => {}}
-          initialFolderName={folder?.name}
-        />
-      </ModalContent>
-    </Modal>
+    <>
+      <Modal
+        modalKey='editfolder'
+        isOpen={isOpen}
+        onClose={handleClose}
+        zIndex={zIndex.CREATE_PLAYLIST_MODAL}
```

Comment:
Reusing this zIndex is probably fine but I did notice that we have a separate `EDIT_TRACK_MODAL` that has the same value of `1000`. So maybe it makes sense to create a `EDIT_PLAYLIST_FOLDER_MODAL`

Reply:
ah! ok thanks for catching that, will do